site/book/05-developing-functions/04-developing-in-Starlark.md

```diff
@@ -0,0 +1,56 @@
+You can write your functions in Starlark script.
+
+?> Starlark SDK is in *experimental* stage. A new version will come up soon.
```

Comment:
May be take out `A new version will come up soon`. Instead, drop a callout for feedback :)

`New version coming soon` may be discouraging :)

Reply:
I see. sure I'll drop that line. I'm not sure if we want to ask for feedback here since we already know how the starlark should be changed.